src/algebra/group_power/basic.lean

```diff
@@ -454,10 +454,10 @@ end
   {a : R} (n : ℕ) (h : a ≠ 0) : a ^ n ≠ 0 :=
 mt pow_eq_zero h
 
-lemma pow_abs [linear_ordered_comm_ring R] (a : R) (n : ℕ) : (abs a)^n = abs (a^n) :=
+lemma pow_abs [linear_ordered_ring R] (a : R) (n : ℕ) : (abs a)^n = abs (a^n) :=
```

Comment:
```suggestion
lemma pow_abs [linear_ordered_ring R] (a : R) (n : ℕ) : abs a ^ n = abs (a ^ n) :=
```

Reply:
I would prefer to preserve parentheses on the left. Anyway,
bors d+

Reply:
```suggestion
lemma pow_abs [linear_ordered_ring R] (a : R) (n : ℕ) : (abs a) ^ n = abs (a ^ n) :=
```